sui-execution/latest/sui-move-natives/src/dynamic_field.rs

```diff
@@ -353,8 +360,14 @@ pub fn remove_child_object(
     let child_id = pop_arg!(args, AccountAddress).into();
     let parent = pop_arg!(args, AccountAddress).into();
     assert!(args.is_empty());
+    // NB: We need to borrow the runtime and grab the mutable reference and then pass this into
+    // `get_or_fetch_object` so that the lifetime of the returned object is tied to the lifetime of
+    // the runtime reference we are creating here and the borrow checker will be happy with us.
+    let object_runtime: &NativeContextMut<'_, ObjectRuntime<'_>> = context.extensions().get();
```

Comment:
```suggestion
    let object_runtime: &NativeContextMut<'_, ObjectRuntime<'_>> = context.extensions().get();
```
Bit of a nit, but let me message you a cleaner way for this. This seems a bit messy